refactor(cli): remove duplicated cli version telemetry data (#27792)
expo/expo: docs/pages/more/expo-cli.mdx

| `EXPO_USE_UNSTABLE_DEBUGGER`        | **boolean** | <StatusTag status="experimental" /><br/>Enable the experimental debugger from React Native.                                                                                                                                                                                                                                                                                          |
| `EXPO_ADB_USER`                     | **string**  | <StatusTag status="SDK 50+" /><br/>Set the `user` number that should be passed to `--user` with ADB commands. Used for installing APKs on Android devices with multiple profiles. Defaults to `0`.                                                                                                                                                                                   |

> | `EXPO_NO_TELEMETRY_DETACH`          | **boolean** | <StatusTag status="SDK 51+" /><br/>Send telemetry events in the main thread of `@expo/cli`. This slows the CLI down, as it has to wait until all events are sent.                                                                                                                                                                                                                    |

Reviewer: ```suggestion
| `EXPO_NO_TELEMETRY_DETACH`          | **boolean** | <StatusTag status="SDK 51+" /><br/>Send telemetry events in the main thread of `@expo/cli`. This causes the CLI to slow down as it waits for all the events to be sent.                                                                                                                                                                        |
```

- Let's update the verbiage here to clearly explain what happens to CLI and why (without adding pause to the sentence).

Author: Ah, this already got merged on the root PR of these stacked PRs https://github.com/expo/expo/pull/27787/files#diff-8c2b6f8e985e42676c1d874b0750a60dba9562499a1728eaea950dfa73e29c4eUBR506

I can change it in a follow up PR though

Reviewer: No problem. Thank you 🙏
